allensdk/brain_observatory/behavior/behavior_session.py

```diff
@@ -1354,6 +1376,40 @@ def _read_data_from_stimulus_file(
                 task_parameters,
                 trials)
 
+    @classmethod
+    def _read_eye_tracking_table(
+            cls,
+            eye_tracking_file: EyeTrackingFile,
+            eye_tracking_metadata_file: EyeTrackingMetadataFile,
+            sync_file: SyncFile,
+            z_threshold: float,
+            dilation_frames: int) -> EyeTrackingTable:
+
+        # this is possible if instantiating from_lims
+        if sync_file is None:
+            msg = ("sync_file is None for this session; "
+                   "do not know how to create an eye tracking "
+                   "table without a sync_file")
+            raise RuntimeError(msg)
+
+        sync_path = pathlib.Path(sync_file.filepath)
+
+        frame_times = sync_utilities.get_synchronized_frame_times(
+            session_sync_file=sync_path,
+            sync_line_label_keys=SyncDataset.EYE_TRACKING_KEYS,
+            drop_frames=None,
+            trim_after_spike=False)
+
+        stimulus_timestamps = StimulusTimestamps(
+                timestamps=frame_times,
```

Comment:
This should be `frame_times.value`. It expects ndarray, not series

Reply:
I have been exercising that code. It worked... anyway: I'll fix it.